src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -2,10 +2,7 @@
 
 import com.google.common.base.Strings;
 import com.joss.conductor.mobile.util.PageUtil;
-import io.appium.java_client.AppiumDriver;
-import io.appium.java_client.CommandExecutionHelper;
-import io.appium.java_client.MobileCommand;
-import io.appium.java_client.TouchAction;
+import io.appium.java_client.*;
```

Comment:
it is best practice to avoid star imports in java classes,https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad